readme.md

```diff
@@ -15,25 +15,25 @@ ___
 ![screenshot](./references/search-bar-example.png)
 
 ### Requirements:
-- use images from [src/images](src/images)
-- there must be two search bars
-- search bar must have width 100%
-- distance between two search bars must be 20px
+- use images from [src/images](src/images) +++
+- there must be two search bars +++
+- search bar must have width 100% +++
+- distance between two search bars must be 20px +++
```

Comment:
Why did you add **+++** ?

Reply:
just a check-list :) to make sure I didn't miss anything from the list